src/main/java/com/jcabi/log/MulticolorLayout.java

```diff
@@ -137,6 +131,11 @@
      */
     private transient String base;
 
+    /**
+     * Helper class to store color data.
```

Comment:
@piddubnyi
> Helper class

Please look at http://blogs.msdn.com/b/nickmalik/archive/2005/09/06/461404.aspx and http://www.smart421.com/application-development/why-helper-singletons-and-utility-classes-are-mostly-bad-2/